### PR TITLE
Render storageclassname only if it exists in PVC template

### PR DESCRIPTION
**Pull request by fabubaker**

### CODE REVIEW

File: helm/rayyanlib/templates/_persistentvolumeclaims.yaml [hunk 1]
```diff
@@ -8,7 +8,9 @@ metadata:
 spec:
   accessModes:
     - {{ $volume.accessModes | default "ReadWriteOnce" }}
+  {{- if $volume.storageClassName }}
   storageClassName: {{ $root.Release.Namespace }}-{{ $root.Release.Namespace }}-{{ $volume.storageClassName }}
```
Comment (hammady) [inline helm/rayyanlib/templates/_persistentvolumeclaims.yaml:12]: This is problematic for storage classes which are not created by rayyanlib templates.

**Review by hammady — CHANGES_REQUESTED** [review]

File: helm/rayyanlib/templates/_persistentvolumeclaims.yaml (hunk 1 above)
Comment (fabubaker) [reply helm/rayyanlib/templates/_persistentvolumeclaims.yaml]: As discussed, the prefix will be removed and the existing PVCs will be replaced.

Comment (hammady) [reply helm/rayyanlib/templates/_persistentvolumeclaims.yaml]: The change as implemented warrants a major version bump I think.

Comment (fabubaker) [reply helm/rayyanlib/templates/_persistentvolumeclaims.yaml]: Yep, sounds fair.

**Review by hammady — APPROVED** [review]

**Pull request merged**